unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDView.cs

```diff
@@ -76,4 +86,9 @@ public void SetVisibility(bool visible)
         else if (!visible && mainShowHideAnimator.isVisible)
             mainShowHideAnimator.Hide();
     }
+
+    private void OnDestroy()
+    {
+        mouseCatcher.OnMouseLock -= OnMouseLocked;
```

Comment:
```suggestion
        mouseCatcher?.OnMouseLock -= OnMouseLocked;
```

We are checking `mouseCatcher` for `null` above, if the value is nullable we have to check it here as well.